fabric-content-registries-v0/src/main/java/net/fabricmc/fabric/impl/content/registry/FuelRegistryImpl.java

```diff
@@ -36,12 +38,26 @@
 	private static final Logger LOGGER = LogManager.getLogger();
 	private final Object2IntMap<ItemConvertible> itemCookTimes = new Object2IntLinkedOpenHashMap<>();
 	private final Object2IntMap<Tag<Item>> tagCookTimes = new Object2IntLinkedOpenHashMap<>();
+	private Object2IntMap<Item> fuelTimeMap;
+	private boolean fuelTimeMapNeedsUpdate = true;
 
-	public FuelRegistryImpl() { }
+	public FuelRegistryImpl() {
+		ServerLifecycleEvents.END_DATA_PACK_RELOAD.register((server, serverResourceManager, success) -> {
```

Comment:
Have you tested if this event is emitted on a client when it receives the tags on game-join from the server?
@i509VCB Since you are the event-master, do you know this off the top of your head?

Reply:
It doesn't, but 824377e addresses this issue!